data/apply-ne_disputed_border_suppression.sql

```diff
@@ -0,0 +1,512 @@
+            fclass_jp = 'unrecognized',
+            fclass_ko = 'unrecognized',
+            fclass_ma = 'unrecognized',
+            fclass_nl = 'unrecognized',
+            fclass_np = 'unrecognized',
+            fclass_pk = 'unrecognized',
+            fclass_pl = 'unrecognized',
+            fclass_ps = 'unrecognized',
+            fclass_pt = 'unrecognized',
+            fclass_ru = 'unrecognized',
+            fclass_sa = 'unrecognized',
+            fclass_se = 'unrecognized',
+            fclass_tr = 'unrecognized',
+            fclass_tw = 'unrecognized',
+            fclass_ua = 'unrecognized',
+            fclass_us = 'unrecognized',
+            fclass_vn = 'unrecognized'
+where l.ne_id in (
+    select ne_id from abkahzia
+    union
+    select ne_id from donbass
+    union
+    select ne_id from nagorno_karabakh
+    union
+    select ne_id from nothern_cyprus
+    union
+    select ne_id from somaliland
+    union
+    select ne_id from south_ossetia
+    union
+    select ne_id from transnistria
+    )),
+
+
+-- Add Russian view to Abkhazia, Donbass, South Ossetia, Transnistria
+ru_ne_10m_admin_0_boundary_lines_land as (
+update ne_10m_admin_0_boundary_lines_land l set
+            fclass_ru = 'country'
+where l.ne_id in (
+    select ne_id from abkahzia
+    union
+    select ne_id from donbass
+    union
+    select ne_id from south_ossetia
+    union
+    select ne_id from transnistria
+)),
+
+ru_ne_50m_admin_0_boundary_lines_land as (
+update ne_50m_admin_0_boundary_lines_land l set
+            fclass_ru = 'country'
+where l.ne_id in (
+    select ne_id from abkahzia
+    union
+    select ne_id from donbass
+    union
+    select ne_id from south_ossetia
+    union
+    select ne_id from transnistria
+    )),
+
+ru_ne_110m_admin_0_boundary_lines_land as (
+update ne_110m_admin_0_boundary_lines_land l set
+            fclass_ru = 'country'
+where l.ne_id in (
+    select ne_id from abkahzia
+    union
+    select ne_id from donbass
+    union
+    select ne_id from south_ossetia
+    union
+    select ne_id from transnistria
+    )),
+
+ru_ne_10m_admin_0_boundary_lines_disputed_areas as (
+update ne_10m_admin_0_boundary_lines_disputed_areas l set
+            fclass_ru = 'country'
+where l.ne_id in (
+    select ne_id from abkahzia
+    union
+    select ne_id from donbass
+    union
+    select ne_id from south_ossetia
+    union
+    select ne_id from transnistria
+    )),
+
+ru_ne_50m_admin_0_boundary_lines_disputed_areas as (
+update ne_50m_admin_0_boundary_lines_disputed_areas l set
+            fclass_ru = 'country'
+where l.ne_id in (
+    select ne_id from abkahzia
+    union
+    select ne_id from donbass
+    union
+    select ne_id from south_ossetia
+    union
+    select ne_id from transnistria
+    )),
+
+-- Add Russian disputed view to Nagorno-Karabakh
+ab_do_so_tr_ne_10m_admin_0_boundary_lines_land as (
+update ne_10m_admin_0_boundary_lines_land l set
+            fclass_ru = ''
```

Comment:
Is the intent this is "default" which happens to be `unrecognized`?

Reply:
Do we want this to be unrecognized? I thought we wanted to show a disputed border there for russia to show that they acknowledge something existing, but not quite a country.